salt/states/boto_secgroup.py

```diff
@@ -388,8 +388,8 @@ def _rules_present(name, rules, vpc_id=None, vpc_name=None,
     3. return 'old' and 'new' group rules
     '''
     ret = {'result': True, 'comment': '', 'changes': {}}
-    sg = __salt__['boto_secgroup.get_config'](name, None, region, key, keyid,
-                                              profile, vpc_id, vpc_name)
+    sg = __salt__['boto_secgroup.get_config'](name=name, group_id=None, region=region, key=key,
+                                              keyid=keyid, profile=profile, vpc_id=vpc_id)
```

Comment:
whoops! nice catch! I don't believe it was there when i made the change to my local a while back.. missed it on the merge from develop :)